src/darker/__main__.py

```diff
@@ -1,4 +1,4 @@
-"""Darker - apply black reformatting to only areas edited since the last commit"""
+"""Darker - re-format only code areas edited since the last commit."""
```

Comment:
nit: could just be format which I think is marginally cleaner

Reply:
Re-wrote that as
>Darker - format only code areas modified between Git revisions.